src/localize/languages/it.json

```diff
@@ -2,83 +2,83 @@
   "$schema": "../schema/schema.json",
   "common": {
     "name": "Scheda Meteoalarm",
-    "description": "La scheda Meteoalarm ti informa degli eventi meteorologici in corso.",
-    "warning": null,
-    "expected": null,
+    "description": "La scheda Meteoalarm ti avvisa degli eventi meteorologici in corso.",
+    "warning": "Attenzione",
+    "expected": "Previsto",
     "unavailable": {
-      "long": null,
-      "short": null
+      "long": "L'integrazione non è disponibile",
+      "short": "Non disponibile"
     }
   },
   "error": {
     "missing_entity": "È necessario specificare un'entità!",
-    "invalid_integration": null,
+    "invalid_integration": "Questa integrazione non è valida!",
     "entity_invalid": {
-      "single": null,
-      "multiple": null
+      "single": "L'integrazione selezionata non corrisponde all'entità selezionata.",
+      "multiple": "L'integrazione selezionata non corrisponde alle entità selezionate: {entity}."
     }
   },
   "editor": {
-    "entity": null,
-    "integration": null,
-    "required": null,
-    "recommended": null,
-    "override_headline": null,
-    "hide_when_no_warning": null,
-    "hide_caption": null,
-    "disable_swiper": null,
+    "entity": "Entità",
+    "integration": "Integrazione",
+    "required": "Richiesto",
+    "recommended": "Consigliato",
+    "override_headline": "Sovrascrivere l'intestazione fornita dall'integrazione",
+    "hide_when_no_warning": "Nascondi quando non ci sono avvisi",
+    "hide_caption": "Nascondi la descrizione",
+    "disable_swiper": "Disabilità lo swipe delle allerte",
     "description": {
-      "start": null,
-      "current_expected": null,
-      "slots": null,
-      "separate_events": null,
-      "warning_watch_statement": null,
-      "end": null
+      "start": "L'integrazione selezionata crea più entità.",
+      "current_expected": "La prima con gli eventi attuali e la seconda con quelli futuri.",
+      "slots": "L'esatta quantità di è determinata dalla sua configurazione.",
+      "separate_events": "Ogni evento ha un'entità propria e separata.",
+      "warning_watch_statement": "Esistono 3 entità: avvertenze, considerazioni e avvisi/messaggi.",
+      "end": "Dovreste aggiungerle tutte per essere sicuro di non perdere nessun avviso."
```

Comment:
Maybe "Dovrest**i** aggiungerle ..." ?

Reply:
Yes this is a typo, sorry, I will correct